### ISSUE TITLE
Zigbee topology

### ISSUE THREAD

**Issue by martin9115** [hobbyquaker/node-red-contrib-zigbee#128]
Hi, Can this libraries help me build zigbee mesh?

**Comment by stuckis6er** [comment]
Yes, I would say.

best regards

**Comment by martin9115** [comment]
Any example?

**Comment by martin9115** [comment]
Any example ?

**Comment by james2mid** [comment]
Zigbee should form a mesh without any additional configuration. The underlying zigbee-herdsman package seems to permit this. I haven't yet been able to configure this with NodeRED yet though, it's quite tricky.

**Comment by martin9115** [comment]
Yes, but i wish to understand the principle (algorithm) of the topology, like star, mesh etc. Any help?

**Comment by james2mid** [comment]
Within the documentation for `zigbee2mqtt`, there is an overview of how the network works. [Check it out](https://www.zigbee2mqtt.io/information/zigbee_network.html). It described the different kinds of nodes and their role in the network.

From what I've read, I believe so long as the devices have been paired onto the network via the coordinator, then the mesh network is self-organising. The coordinator just needs to send a message through its connected devices and it will be routed to the recipient node.

Hope that helps :)
